packages/react/src/schematics/application/files/app/src/app/__fileName__.tsx__tmpl__

```diff
@@ -252,12 +255,12 @@ export class App extends Component {
 <% } else { %>
 export const App = () => {
   /*
-   * Replace the elements below with your own.
+   * Replace the elements below with your own. <% if (hasStyles) { %>
    *
-   * Note: The corresponding styles are in the ./<%= fileName %>.<%= style %> file.
+   * Note: The corresponding styles are in the ./<%= fileName %>.<%= style %> file. <% } %> 
    */
   return (
-    <<%= wrapper %><% if (!styledModule) {%> className="app"<% } %>>
+    <<%= wrapper %><% if (!styledModule && hasStyles) {%> className="app"<% } %>>
       <%= innerJsx %>
     </<%= wrapper %>>
   );
```

Comment:
@jaysoo just an FYI - if you generate an app with no styles, the default page doesn't look so good anymore 😄 let me know if you think it's worth spending the time to simplify it

since it's just a placeholder that people will most likely remove quickly, I assumed it's not worth generating different placeholders for no styles

Reply:
I think that's fine since the user selected `none` they shouldn't expect anything to be styled.